elastic-beats/templates/metricbeat.yml.j2

```diff
@@ -0,0 +1,94 @@
+########################## Metricbeat Configuration ###########################
+# You can find the full configuration reference here:
+# https://www.elastic.co/guide/en/beats/metricbeat/index.html
+
+#==========================  Modules configuration ============================
+#------------------------------- Input -------------------------------
+metricbeat:
+  modules:
+    - module: system
+      metricsets:
+        - cpu
+        - load
+        - filesystem
+        - fsstat
+        - memory
+        - network
+        - process
+        - socket
+      enabled: true
+      period: 10s
+      processes: ['.*']
+
+    #------------------------------- Docker Module -------------------------------
+    #- module: docker
```

Comment:
is this not going to replace the dockbeats.  so should be enabled in the template